src/Utils.php

```diff
@@ -107,17 +107,7 @@ public static function chooseHandler(): callable
      */
     public static function defaultUserAgent(): string
     {
-        static $defaultAgent = '';
-
-        if (!$defaultAgent) {
-            $defaultAgent = 'GuzzleHttp/Guzzle';
-            if (\extension_loaded('curl') && \function_exists('curl_version')) {
-                $defaultAgent .= ' curl/' . \curl_version()['version'];
-            }
-            $defaultAgent .= ' PHP/' . PHP_VERSION;
-        }
-
-        return $defaultAgent;
+        return 'GuzzleHttp/Guzzle';
```

Comment:
Should be GuzzleHttp only as the part behind the slash is specified as a version in the corresponding RFC.

Reply:
It doesn't matter too much, does it?

I named the guzzle 7 to "Guzzle", and now there is a version. =)

Reply:
Depends as always 😁. The RFC defines the version as a Token. And the agent is a token too. So it is may be conform but unexpected as the examples show numeric version strings.

But what the RFC clearly states is that the version part itself is optional.

Reply:
>But what the RFC clearly states is that the version part itself is optional.

Okey, I've not read the RFC. Im happy to drop this. 

Thank you